addons/website/models/ir_module_module.py

```diff
@@ -730,6 +730,8 @@ def get_distinct_snippet_names(structure):
             # Invoke for themes and website_* - otherwise on -u website, the
             # additional primary snippets they require are deleted by _process_end.
             for module in self.env['ir.module.module'].search([
+                '|',
+                ('name', '=', 'theme_default'),
```

Comment:
Needed because theme_default is still used as a "default theme value" in some places
https://github.com/odoo/odoo/commit/61f71de42c96a0ba556753fbb8bc70c10928264f#diff-5e92e473fa4d3da6db7ef727fb217dad51ef6c2383913edca73fe040a23e82c2R299

Probably should see one day to remove it, doesn't seems to bring much value nowadays?